### PR TITLE
A new Integration with NozomiGuardian Integration

### PR DESCRIPTION
**Pull request by ayman-m**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A new Integration with NozomiGuardian, an OT network visibility solution

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/NozomiNetworks/Integrations/NozomiGuardian/NozomiGuardian.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            raws.append(raw_response)
+            nozomi_ec.append({
+                'QueryResults': raws
+            })
+
+        if not raws:
+            return_error("%s - Could not find any results for given query" % INTEGRATION_NAME)
+
+        context_entry = {
+            "NozomiGuardian": {"Queries": nozomi_ec}
```
Comment (yaakovi) [inline Packs/NozomiNetworks/Integrations/NozomiGuardian/NozomiGuardian.py:64]: why do you need so many layers? `NozomiGuardian.Queries.QueryResults`?
better to have: `NozomiGuardian.QueryResults`

**Comment by ayman-m** [comment]
@yaakovi I did push a new commit to reflect the above changes

File: Packs/NozomiNetworks/Integrations/NozomiGuardian/NozomiGuardian.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    params = demisto.params()
+    username = params.get('credentials', {}).get('identifier', '')
+    password = params.get('credentials', {}).get('password', '')
+    base_url = params['url'][:-1] if (params['url'] and params['url'].endswith('/')) else params['url']
```
[the remark this answers is not in the document]
Comment (yaakovi) [reply Packs/NozomiNetworks/Integrations/NozomiGuardian/NozomiGuardian.py:131]: you missed this one.

**Comment by kirbles19** [comment]
@yaakovi Done.

**Pull request closed without merging**